src/scss/components/optional/_badges.scss

```diff
@@ -23,9 +23,15 @@ $badge-important-color      : $white;
 $badge-important-bgColor    : $orange;
 $badge-award-color          : $purple;
 $badge-award-bgColor        : $purple--light;
+$badge-ok-color             : $white;
+$badge-ok-bgColor           : $green;
+$badge-dark-color           : $white;
```

Comment:
Do you need to specify this one? Line 26 could be used instead?

Reply:
but that variable is for the OK button?

Reply:
OK. So a more generic one would be better so we don't end up with loads of variables pointing to the same color. but it's fine.

Reply:
I'm for this as long as we don't end up with hundreds of badge types in the future (consolidation then would be good). I guess if that happens though we can optimise it then as there's still only a handful of types.